cms/articles/models.py

```diff
@@ -69,12 +71,21 @@ def latest_release(self, request: "HttpRequest") -> "TemplateResponse":
 
     @path("previous-releases/")
     def previous_releases(self, request: "HttpRequest") -> "TemplateResponse":
-        response: TemplateResponse = self.render(
-            request,
-            # TODO: update to include drafts when looking at previews holistically.
-            context_overrides={"pages": StatisticalArticlePage.objects.live().child_of(self).order_by("-release_date")},
-            template="templates/pages/statistical_article_page--previous-releases.html",
-        )
+        children = StatisticalArticlePage.objects.live().child_of(self).order_by("-release_date")
+        paginator = Paginator(children, per_page=settings.PREVIOUS_RELEASES_PER_PAGE)
+
+        try:
+            pages = paginator.page(int(request.GET.get("page", 1)))
```

Comment:
`paginator` will raise a `PageNotAnInteger` exception, so you can remove the `int()` and change `ValueError` below to `PageNotAnInteger`. This is because `.page()` will call `int()` for you.

https://docs.djangoproject.com/en/5.1/ref/paginator/#django.core.paginator.Paginator.page

Reply:
Thank you good suggestion

Reply:
@MaciekBaron can we resolve this now? if you're you happy with the changes? Thanks =)